### PR TITLE
feat(highlights): Add the option to set the groups for highlights

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: client/app/scripts/superdesk-highlights/highlights.js [hunk 1]
```diff
@@ -79,6 +79,21 @@
                 highlight: highlight._id
             };
 
+            if (highlight.groups && highlight.groups.length > 0) {
+                var groups = [{
```
Comment (amagdas) [inline client/app/scripts/superdesk-highlights/highlights.js]: This part looks to be a copy of the packagesService, any reason why we can't reuse that?

File: client/app/scripts/superdesk-highlights/highlights.js [hunk 2]
```diff
@@ -323,6 +350,51 @@
...
+            if (group !== '') {
+                $scope.editingGroup = {'name': group, 'id': $scope.configEdit.groups.indexOf(group) + 1};
+            } else {
+                $scope.editingGroup = {'name': '', 'id': 0};
+            }
+        };
+
+        $scope.removeGroup = function(group) {
+            $scope.configEdit.groups.splice($scope.configEdit.groups.indexOf(group), 1);
+            console.log('after remove: ', $scope.configEdit.groups);
```
Comment (amagdas) [inline client/app/scripts/superdesk-highlights/highlights.js]: probably not needed.

File: client/app/scripts/superdesk-packaging/packaging.js [hunk 3]
```diff
@@ -217,7 +217,17 @@
 
         $scope.$watch('item', function(item) {
             $scope.highlight = item.highlight;
-            init = true;
+            if ($scope.highligh) {
```
Comment (amagdas) [inline client/app/scripts/superdesk-packaging/packaging.js]: `$scope.highlight` instead of `$scope.highligh`

**Comment by amagdas** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**